README.md

```diff
@@ -340,3 +340,12 @@ The suggested permission policies for AWS S3 are:
 - `s3:List*`
 
 If you do not want to provide `region` in the configuration, you should enable `s3:GetBucketLocation` permission policy as well. 
+
+## Troubleshooting
```

Comment:
I had an idea
Not sure about permissions and stuff hence an idea
What if, upon sink deploy, we check the broker config and validate maxUnacked > batchSize?
Because we said if it’s smaller, it’s an anti-pattern
Is there a code that is running only upon deployment of a sink?
Or maybe upon init, check that somehow?
If sink can access PulsarAdmin?

Reply:
While Pulsar functions have access to a `PulsarAdmin` instance in their `Context`, sinks and source connectors unfortunately do not. Instantiating it ourselves would mean the connector would need additional configuration parameters for talking to the broker, which feels clunky. I will open a proposal to add `PulsarAdmin` to the `SinkContext` and `SourceContext` and merge this doc change for the time being.